lms/djangoapps/courseware/access.py

```diff
@@ -270,17 +270,22 @@ def _can_enroll_courselike(user, courselike):
     return ACCESS_DENIED
 
 
-def _has_access_course_desc(user, action, course):
+def _has_access_courselike(user, action, courselike):
     """
-    Check if user has access to a course descriptor.
+    Check if user has access to a course.
+
+    Arguments:
+        user (User): the user whose course access we are checking.
+        action (string): The action that is being checked.
+        courselike (CourseDescriptor or CourseOverview): The object
+            representing the course that the user wants to access.
```

Comment:
Blocks of type "library_content" are of type `XBlock` or `XModule`, right?  So wouldn't they be handled through another path in `has_access` ([L116](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/access.py#L116))?  And actually call `_has_access_descriptor` instead?